### PR TITLE
Fixing Tape Recorder Voices

### PR DESCRIPTION
**Pull request by Sharkmare**
Carbon/Human has a voice variable for voice changers and to not go unknown when the face is covered without ID, tape recorder now respects that.

### CODE REVIEW

File: code/game/objects/items/devices/taperecorder.dm [hunk 1]
```diff
@@ -91,8 +91,16 @@
...
 /obj/item/taperecorder/hear_talk(mob/M, list/message_pieces, verb)
 	var/msg = multilingual_to_message(message_pieces, requires_machine_understands = TRUE, with_capitalization = TRUE)
+	var/voice = "Unknown"
+	if (M.type == /mob/living/carbon/human)
+	{
+		var/mob/living/carbon/human/H = M
+		voice = H.voice
+	}
+	else
+		voice = M.name
```
Comment (Atermonera) [inline code/game/objects/items/devices/taperecorder.dm:101]:
```suggestion
	var/voice = M.name ? M.name : "Unknown"
	if (ishuman(M))
		var/mob/living/carbon/human/H = M
		voice = H.voice
```
DM is non-context-free, you don't need the curly braces.

Comment (Atermonera) [reply code/game/objects/items/devices/taperecorder.dm:101]: @Sharkmare bumping this, you gonna get back to this?

Comment (Sharkmare) [reply code/game/objects/items/devices/taperecorder.dm:101]: for some reason i only got the bump as a notification odd

**Review by Atermonera — DISMISSED** [review]

**Comment by Sharkmare** [comment]
Apparently i made a mistake somewhere down the road and also found that mobs do have a voice var we can just use that is also use on carbons according to say code

**Comment by Atermonera** [comment]
I don't think `voice_name` is the var you want to use here. A cursory search of code shows that it's just set to "unkown" for all human mobs, meaning everyone will show up as "unknown" in the transcript.

**Comment by Sharkmare** [comment]
> I don't think `voice_name` is the var you want to use here. A cursory search of code shows that it's just set to "unkown" for all human mobs, meaning everyone will show up as "unknown" in the transcript.

Good point, GetVoice() is probably the correct way to handle this.

**Review by Atermonera — APPROVED** [review]

**Pull request merged**